modules/services/window-managers/hypr.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  inherit (lib) hm types mkEnableOption mkIf mkOption mkPackageOption platforms;
```

Comment:
Hello, @rycee !

Can I use this more explicit listing of auxiliary `lib` functions?